src/models/BoardGame.py

```diff
@@ -29,3 +29,7 @@ def __post_init__(self):
         else:
             object.__setattr__(self, "overallrank", float(self.overallrank))
         # This bypasses frozen=true by modifying the object class which all objects inherent from including dataclasses
+
+    # BoardGame object must be subscriptable in order to preform indexing on object's attributes
+    def __getitem__(self, item):
```

Comment:
curious: why are you indexing on the attribute? where is not possible to just do attribute accessing directly?

Reply:
Hmmmm, it looks like I'm not erroring anymore without this so I can probably remove it.  I think it was needed as part of some earlier attempted implementation, but I had some erroring (maybe it was sorting a list of BoardGames) but it doesn't seem to be an issue anymore.